dashboard/src/reducers/quisbyChartReducer.js

```diff
@@ -0,0 +1,38 @@
+import * as TYPES from "../actions/types";
+
+const initialState = {
+  data: [],
+  chartData: [],
+  activeResourceId: "",
+  unsupportedType: "",
+};
+
+const QuisbyChartReducer = (state = initialState, action = {}) => {
+  const { type, payload } = action;
+  switch (type) {
+    case TYPES.GET_QUISBY_DATA:
+      return {
+        ...state,
+        data: payload,
+      };
```

Comment:
Shouldn't this action be named `SET...` instead of `GET...`?

Reply:
Renamed